common/versionConfig.csv

```diff
@@ -0,0 +1,5 @@
+com.microsoft.azure.v3,autorest-clientruntime-for-java,2.0.0,2.0.1-SNAPSHOT
```

Comment:
We should understand if anything will be using this file and if not remove as part of clean-up.